<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertstudent</class>
 <widget class="QWidget" name="insertstudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1409</width>
    <height>983</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>901</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>USN</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="usnEntry"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="nameEntry"/>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Branch</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="branchEntry"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Fee Status</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="paidToggle">
        <property name="text">
         <string>Paid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="notPaidToggle">
        <property name="text">
         <string>Not Paid</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="submit">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="errorFlag">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Student Record</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
